planetiler-core/src/test/java/com/onthegomap/planetiler/PlanetilerTests.java

```diff
@@ -1661,6 +1663,7 @@ public void processFeature(SourceFeature source, FeatureCollector features) {
       .addOsmSource("osm", tempOsm)
       .addNaturalEarthSource("ne", TestUtils.pathToResource("natural_earth_vector.sqlite"))
       .addShapefileSource("shapefile", TestUtils.pathToResource("shapefile.zip"))
+      .addShapefileDirectorySource("shapefiledir", TestUtils.pathToResource(""), "shapefile*.zip")
```

Comment:
Is the intention here to include that same file as the line above just through glob matching? If so, you could add another variation to the ValueSource arguments of the test to switch whether this runs with `addShapefileSource` or `addShapefileDirectorySource` to make sure they both produce the same results?

Reply:
Good idea, added a more explicit test.